src/features/game/expansion/Land.tsx

```diff
@@ -347,6 +377,8 @@ export const Land: React.FC = () => {
     scrollIntoView(Section.GenesisBlock, "auto");
   }, []);
 
+  const isFirstRender = useFirstRender();
```

Comment:
The desire is to have mushrooms _grow_ from the ground if the player is half way through a session. Just a neat little feature.

This hook allows the first Mushrooms to skip that animation otherwise it looks strange having multiple mushrooms grow every refresh.